<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    
    <title>Research | Microbial Systems</title>
    <meta name="viewport" content="width=device-width,minimum-scale=1">
    <meta name="generator" content="Hugo 0.75.1" />
    
    
      <META NAME="ROBOTS" CONTENT="INDEX, FOLLOW">
    

    
    
      <link href="/dist/css/app.4fc0b62e4b82c997bb0041217cd6b979.css" rel="stylesheet">
    

    

    
      
<link rel="shortcut icon" href="/media/logo.png" type="image/x-icon" />


    

    
    
      <link href="/research/index.xml" rel="alternate" type="application/rss+xml" title="Microbial Systems" />
      <link href="/research/index.xml" rel="feed" type="application/rss+xml" title="Microbial Systems" />
      
    
    
    <meta property="og:title" content="Research" />
<meta property="og:description" content="The Miller Lab at the University of Colorado Denver" />
<meta property="og:type" content="website" />
<meta property="og:url" content="https://microbial.systems/research/" />
<meta property="og:updated_time" content="2020-09-25T22:07:47-06:00" />
<meta itemprop="name" content="Research">
<meta itemprop="description" content="The Miller Lab at the University of Colorado Denver">
<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="Research"/>
<meta name="twitter:description" content="The Miller Lab at the University of Colorado Denver"/>

      
    
	
  </head>

  <body class="ma0 avenir bg-near-white production">

    

  
  
  <header class="cover bg-top" style="background-image: url('https://microbial.systems/media/2018_GBE_figure_3_small.png');">
    <div class="bg-black-60">
      <nav class="pv3 ph3 ph4-ns" role="navigation">
  <div class="flex-l justify-between items-center center">
    <a href="/" class="f3 fw2 hover-white no-underline white-90 dib">
      
        Microbial Systems
      
    </a>
    <div class="flex-l items-center">
      

      
        <ul class="pl0 mr3">
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white no-underline white-90" href="/lab-news/" title="Lab News page">
              Lab News
            </a>
          </li>
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white no-underline white-90" href="/research/" title="Research page">
              Research
            </a>
          </li>
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white no-underline white-90" href="/lab-members/" title="Lab Members page">
              Lab Members
            </a>
          </li>
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white no-underline white-90" href="/contact/" title="Contact page">
              Contact
            </a>
          </li>
          
        </ul>
      
      















    </div>
  </div>
</nav>

      <div class="tc-l pv4 pv6-l ph3 ph4-ns">
        <h1 class="f2 f-subheadline-l fw2 white-90 mb0 lh-title">
          Research
        </h1>
        
      </div>
    </div>
  </header>


    <main class="pb7" role="main">
      
  <article class="pa3 pa4-ns nested-copy-line-height nested-img">
    <section class="cf ph3 ph5-l pv3 pv4-l f4 center measure-wide lh-copy mid-gray"><h2 id="we-are-interested-in-developing-and-utilizing-new-experimental-and-computational-methods-in-order-to-understand-microbial-communities">We are interested in developing and utilizing new experimental and computational methods in order to understand microbial communities.</h2>
<h3 id="link-to-publicationshttpsscholargooglecomcitationshlenuser01itl5iaaaajview_oplist_workssortbypubdate"><a href="https://scholar.google.com/citations?hl=en&amp;user=01ITL5IAAAAJ&amp;view_op=list_works&amp;sortby=pubdate"><strong>Link to publications</strong></a></h3>
<p>Broadly, we are interested in the following areas of research:</p>
<ul>
<li>developing and using experimental and <strong>bioinformatics</strong> tools that better characterize microbial diversity from high-throughput sequencing data</li>
<li>understanding the genomic determinants of <strong>ecosystem-scale processes</strong> within microbial communities</li>
<li>understanding <strong>genomic diversity, molecular biology, and evolutionary history</strong> of microbes from shotgun metagenomic sequencing</li>
</ul>
<p>We have a particular focus on <strong>archaeal diversity and the role of archaea</strong> in microbial communities.  We work with a variety of world-class collaborators on specific experimental systems of interest.  Some of these projects are listed below.  Please also read our publications for more information.</p>
<hr>
<h3 id="determinants-of-microbial-contributions-to-methane-emissions-in-freshwater-wetlands">Determinants of microbial contributions to methane emissions in freshwater wetlands</h3>
<p>Of the three most potent greenhouse gases, methane emissions are the most difficult to predict. Despite being overall carbon sinks, wetlands represent the largest single source of atmospheric methane.  Yet, relatively little is known about the microbial taxa and pathways responsible for methane flux in freshwater wetlands.  Together with the Wrighton lab at Colorado State University and many other great collaborators, we are generating a high-resolution census of freshwater wetland microbial communities, reconstructing the major metabolisms of microbial communities in freshwater wetlands sediments, and aim to understand and ultimately predict how these communities and metabolisms respond to changing climate, hydrology, and variations in available carbon.  This project is producing a large amount of marker gene, metagenomic, and transcriptomic sequencing across all domains of life.  We are working on methods to simultaneously integrate disparate data types from Bacteria, Fungi, Viruses, and especially the understudied Archaea, in order to understand and predict ecosystem-scale processes in freshwater wetlands.</p>
<p><em>Collaborators:</em> <a href="https://wrightonlab.com/">Kelly Wrighton</a>, Colorado State University; <a href="https://ceg.osu.edu/people/bohrer.17">Gil Bohrer</a>, Ohio State University; and many others!</p>
<hr>
<h3 id="improved-bioinformatics-methods-for-inference-of-microbial-protein-function-metabolisms-and-traits">Improved bioinformatics methods for inference of microbial protein function, metabolisms, and traits</h3>
<p>Advances in DNA sequencing and associated genome-enabled high-throughput technologies have made the assembly of microbial genomes and partial genomes recovered from the environment routine.  In theory, computational inference of the protein products encoded by these genomes, and the associated biochemical functions, should allow for the accurate prediction and modeling of key microbial traits, organismal interactions, and ecosystem processes that drive biogeochemical cycles.  In practice, however, a lack of scalable computational annotation tools means these outcomes are rarely achieved without expert manual curation that scales extremely poorly.  We are developing scalable annotation tools to better predict protein function, and better understand and model microbial community metabolism.</p>
<p><em>Collaborators:</em> <a href="https://engineering.ucdenver.edu/academics/departments/computer-science-and-engineering/CSCI-research/data-science-big-data-management-and-mining/research-faculty/farnoush-banaei-kashani">Farnoush Banaei-Kashani</a>, University of Colorado Denver; <a href="https://wrightonlab.com/">Kelly Wrighton</a>, Colorado State University; <a href="https://www.anl.gov/profile/christopher-s-henry">Chris Henry</a>, Argonne National Lab</p>
<hr>
<h3 id="the-golden-retriever-microbiome">The Golden Retriever Microbiome</h3>
<p><a href="https://www.morrisanimalfoundation.org/golden-retriever-lifetime-study">The Golden Retriever Lifetime Study</a> is following  more than 3,000 golden retrievers throughout their lifetimes, mining a large amount of health, environmental and behavior data for associations with canine disease, especially cancer.  We are partnering with the Morris Animal Foundation to collect data on the canine fecal microbiome, and analyzing this data in the context of the larger study.</p>
<p><em>Collaborators:</em> <a href="https://www.morrisanimalfoundation.org/">Morris Animal Foundation</a></p>
<hr>
<h3 id="the-role-of-the-microbiome-in-modulating-effects-of-environmental-pollutant-exposure">The role of the microbiome in modulating effects of environmental pollutant exposure</h3>
<p>Many toxic compounds enter the environment from wastewater, legacy waste sites, and other sources.  These compounds have the potential to alter the gut microbiome in fish, other wildlife, companion animals, and humans.  The nature of these alterations and the potential interactions with host health are poorly understood.  Using high-throughput DNA and RNA sequencing, we are attempting to characterize pollutant-mediated disruption in the fish gut microbiome, and to integrate this data with other complex omics and chemical data.</p>
<p><em>Collaborators:</em> <a href="https://clas.ucdenver.edu/integrative-biology/alan-vajda">Alan Vajda</a>, CU Denver</p>
<hr>
<h3 id="archaeal-diversity-and-evolution">Archaeal diversity and evolution</h3>
<p>New sequencing and computational approaches have allowed us to begin to better understand the vast biodiversity, ecosystem roles, and evolutionary history of archaea.  We are interested in using amplicon and shotgun metagenomic sequencing data to shed light on the still relatively understudied archaea.  Check out our recent paper,
“Complex evolutionary history of translation Elongation Factor 2 and diphthamide biosynthesis in Archaea and parabasalids,”](<a href="https://academic.oup.com/gbe/article/10/9/2380/5060531">https://academic.oup.com/gbe/article/10/9/2380/5060531</a>) for an example of the kinds of questions that deep sequencing and comparative genomics can answer.</p>
<p><em>Collaborators:</em> Thijs Ettema,  Uppsala University; Anja Spang, NIOZ, Royal Netherlands Institute for Sea Research; Brett Baker, UT Austin</p>
</section>
    <section class="flex-ns flex-wrap justify-around mt5">
      
    </section>

</article>

    </main>
    <footer class="bg-black bottom-0 w-100 pa3" role="contentinfo">
  <div class="flex justify-between">
  <a class="f6 fw4 hover-white no-underline white-70 dn dib-ns pv2 ph3" href="https://microbial.systems" >
    &copy;  The Miller Lab 2020 
  </a>
    <div>














</div>
  </div>
</footer>

    

  <script src="/dist/js/app.3fc0f988d21662902933.js"></script>


  </body>
</html>
